src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -330,10 +330,20 @@ private async Task WriteForAllLocalDatabases(ZipArchive archive, JsonOperationCo
                 {
                     using (var rawRecord = ServerStore.Cluster.ReadRawDatabaseRecord(context, databaseName))
                     {
-                        if (rawRecord == null ||
-                            rawRecord.Topology.RelevantFor(ServerStore.NodeTag) == false)
+                        if (rawRecord == null)
                             continue;
 
+                        if (rawRecord.IsSharded)
+                        {
+                            if (rawRecord.Sharding.Orchestrator.Topology.RelevantFor(ServerStore.NodeTag) == false)
```

Comment:
what if the orchestrator is only on node A, but the shards are on B & C, will I get the debug package from nodes A, B & C? or only from node A?

Reply:
from A, B & C

Reply:
from A, B & C